### ISSUE TITLE
[0.11] `get_url` fails for sass files

### ISSUE THREAD

**Issue by apiraino** [getzola/zola#1054]
## Environment

Zola version: 0.11

## Expected Behavior
SASS files should be compiled, placed in the `public` dir and loaded

## Current Behavior
SASS files are compiled, placed in the `public` dir but the when I load them with `get_url` I get an error (see below)

Without bisecting I suspect that this [commit](https://github.com/getzola/zola/commit/36ec33f0429a9f85247a6f76cf4bf176270285d7) could be related.

## Step to reproduce

I have a template (`post.html`) with this CSS directive
`<link rel="stylesheet" href="{{ get_url(path='css/style.css', cachebust=true) | safe }}">`

I have a file in `./sass/css/style.scss` (this file is taken from a Jekyll theme (so it's an import of another sass).
The sass is compiled and I see the `public/css/style.css` minified.

But then `zola serve` return this error:
```
$ RUST_BACKTRACE=1 zola serve
Building site...
-> Creating 67 pages (0 orphan), 0 sections, and processing 0 images
Error: Failed to render page '/home/user/myblog/content/some-page.md'
Reason: Failed to render 'post.html'
Reason: Could not read file `css/style.css`. Expected location: /home/user/myblog/content/css/style.css
```

What do you think? Am I missing something?

Again, thaks for Zola :)

**Comment by Keats** [comment]
Remove the cachebust parameter for now, there is a bug in it.

**Comment by apiraino** [comment]
Bug seems to be fixed in v0.12 - thanks!
